src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -79,7 +108,10 @@ def update_data(self):
                 mibs.logger.error("SyncD 'ASIC_DB' includes invalid FDB_ENTRY '{}': {}.".format(fdb_str, e))
                 continue
 
-            ent = Namespace.dbs_get_all(self.db_conn, mibs.ASIC_DB, s, blocking=True)
+            try:
```

Comment:
Why this change? if blocking=false, why try catch is required?

Reply:
Just to continue with next iteration in case if we get any exception